src/osd/scrubber/pg_scrubber.cc

```diff
@@ -2655,6 +2686,22 @@ void ReplicaReservations::handle_reserve_grant(OpRequestRef op, pg_shard_t from)
     dout(10) << __func__ << ": osd." << from << " scrub reserve = success"
 	     << dendl;
     m_reserved_peers.push_back(from);
+
+    // was this response late?
+    auto now_is = clock::now();
+    if (m_timeout_point && (now_is > *m_timeout_point)) {
+      m_osds->clog->warn() << fmt::format(
+	"osd.{} scrubber pg[{}]: late reservation from osd.{}",
+	m_osds->whoami,
+	m_pg->pg_id,
+	from);
+      dout(10) << fmt::format(" late reservation from osd.{}", from) << dendl;
+      m_timeout_point.reset();
```

Comment:
IIRC, clog warn will also dout to the osd log.

Reply:
Agree (albeit in a somewhat different format). Removed.